v19.2/architecture/sql-layer.md

```diff
@@ -78,6 +78,10 @@ Components of the physical plan are sent to one or more nodes for execution. On
 
 Each processor uses an encoded form for the scalar values manipulated by the query. This is a binary form which is different from that used in SQL. So the values listed in the SQL query must be encoded, and the data communicated between logical processors, and read from disk, must be decoded before it is sent back to the SQL client.
 
+<span class="version-tag">New in v19.2:</span> CockroachDB supports column-oriented ("vectorized") query execution on all [supported SQL operations](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html). Column-oriented execution models contrast with row-oriented execution models, which can offer good performance for [Online transaction processing (OLTP)](https://en.wikipedia.org/wiki/Online_transaction_processing) queries, but suboptimal performance for [Online analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing).
```

Comment:
In general, I think this paragraph is a too "light" on technical detail for the architecture docs.  I would like to see the technical content from `vectorized-execution.html#how-vectorized-execution-works` woven in here.  The architecture docs are explicitly for users who want to know "how stuff works".  Happy to chat more offline about this, I know weaving stuff into the architecture docs can be hard.
I think the text "supports vectorized .. on all supported SQL operations" is slightly misleading from a user POV, since the "Vectorized Query Execution" page then goes on to list all sorts of contexts in which it does not run stuff in vectorized like "streaming operations" etc etc.  It also reads phrasing-wise as a little tautological, "we support all the supported things". Not sure how to suggest a rephrasing in a way that makes sense, since I don't have much technical context here.  Maybe something like "the vectorized engine is enabled for many SQL queries that would benefit from columnarized execution", if that is in fact the case?  Happy to talk offline also if that would be useful.
I would way prefer the link texts here were just "OLTP" and "OLAP".  I think if you are this deep in the SQL layer of the architecture docs, you probably know what those things are.  Also, the linked wiki pages make it clear what they are anyway, whereas "online transaction processing" is still pretty opaque unless you already know what that is.
Further, I think we should link use of "column-oriented execution models" to something like https://en.wikipedia.org/wiki/Column-oriented_DBMS#Column-oriented_systems, and use of "row-oriented" to something like https://en.wikipedia.org/wiki/Column-oriented_DBMS#Row-oriented_systems.
I think the multi-clause sentence "Column-oriented execution models contrast with ..." is kinda hard to follow the logic of.  It should be broken into shorter, more declarative sentences. E.g., "Row-oriented execution models can offer good performance on [OLTP]() queries, but may not perform well on [OLAP]() queries.  Therefore we've added support for columnar ("vectorized" execution, which dramatically improves performance for OLAP ..."